itwm_example/mco/space_sampling/space_samplers.py

```diff
@@ -0,0 +1,158 @@
+import abc
+import numpy as np
+
+from traits.api import ABCHasStrictTraits, Bool, ListFloat
+from force_bdss.api import PositiveInt
+
+
+def convert_samples_pp_to_samples_total(space_dimension, nof_points):
```

Comment:
A doc string here would be useful and could allow us simplify the function name.